src/utils/worker/createResponseListener.ts

```diff
@@ -25,7 +25,7 @@ export function createResponseListener(context: SetupWorkerInternalContext) {
       return
     }
 
-    const response = new Response(responseJson.body, responseJson)
+    const response = new Response(responseJson.body || null, responseJson)
```

Comment:
This is rather brave, but when handling a `Response` instance there is no difference between working with a `new Response(null)` and `new Response('')` in terms of the operable response body (i.e. `.json`/`.text`/etc.).